test/unit/vdom/skip.js

```diff
@@ -2,26 +2,69 @@ import { define, prop, props, symbols, vdom } from '../../../src/index';
 import afterMutations from '../../lib/after-mutations';
 import fixture from '../../lib/fixture';
 
+const { text, elementOpen, elementClose, elementOpenStart, elementOpenEnd, elementVoid } = vdom;
+const sr = el => el[symbols.shadowRoot];
+
 describe('vdom/skip', () => {
   it('should skip the element children', done => {
     const Elem = define('x-test', {
-      props: {
-        num: prop.number()
+      props: { 
+        num: prop.number() 
       },
+      /* eslint indent: 0 */
       render() {
-        vdom.element('div', { skip: true }, () => {
-          vdom.element('span', () => {
-            vdom.text('text');
-          });
-        });
+        text('1 ');
+        elementOpen('div');
+          text('2 ');
+          elementVoid('void');
+          elementOpenStart('span'); elementOpenEnd();
+            text('3 ');
+          elementClose('span');
+          elementOpen('div');
+            text('4 ');
+            elementOpen('span');
+              text('5 ');
+            elementClose('div');
+          elementClose('div');
+        elementClose('div');
+        text('6 ');
+        elementOpen('div', null, null, 'skip', true);
+          text('7 ');
+          elementVoid('void');
+          elementOpenStart('span'); elementOpenEnd();
+            text('8 ');
+          elementClose('span');
+          elementOpen('div');
+            text('9 ');
+            elementOpen('span');
+              text('10 ');
+            elementClose('span');
+          elementClose('div');
+        elementClose('div');
+        text('11 ');
+        elementOpen('div');
+          text('12 ');
+          elementVoid('void');
+          elementOpenStart('span'); elementOpenEnd();
+            text('13 ');
+          elementClose('span');
+          elementOpen('div');
+            text('14 ');
+            elementOpen('span');
+              text('15');
+            elementClose('span');
+          elementClose('div');
+        elementClose('div');
       },
     });
```

Comment:
What about the situation 'a custom element inside a custom element'? Should `skip` affect another custom element's behavior?
What should happen here:
```
const Elem2 = define('x-test2', {
      render() {
        vdom.element('div', () => {
          vdom.text('2');
          vdom.element('slot');
        });
      },
    });
    const Elem = define('x-test', {
      render() {
        vdom.element('x-test2', {skip: true}, () => {
          vdom.element('div', () => {
            vdom.text('1');
          });
        });
      },
    });
```
Right now '2' and 'slot' are rendering despite `{skip: true}` for the x-test2.